enhancements/ingress/configurable-dns-loadbalancerservice.md

```diff
@@ -0,0 +1,316 @@
+
+```go
+// LoadBalancerStrategy holds parameters for a load balancer.
+type LoadBalancerStrategy struct {
+    // <snip>
+
+    // dns indicates if the lifecyle of the wildcard DNS record
+    // associated with the load balancer service will be managed by
+    // the ingress operator.
+    //
+    // +kubebuilder:default:="Managed"
+    // +kubebuilder:validation:Optional
+    // +kubebuilder:validation:Enum=Managed;Unmanaged
+    // +optional
+    DNS LoadBalancerDNSManagementPolicy `json:"dns"`
+}
+
+// LoadBalancerDNSManagementPolicy is a policy for configuring how
+// ingresscontrollers manage DNS.
+//
+// +kubebuilder:validation:Enum=Managed;Unmanaged
+type LoadBalancerDNSManagementPolicy string
+
+const (
+    // ManagedLoadBalancerDNS configures the ingresscontroller to manage
+    // the wildcard DNS records associated with it.
+    ManagedLoadBalancerDNS LoadBalancerDNSManagementPolicy = "Managed"
+    // UnmanagedLoadBalancerDNS configures the ingresscontroller to not
+    // manage the wildcard DNS records associated with it.
+    UnmanagedLoadBalancerDNS LoadBalancerDNSManagementPolicy = "Unmanaged"
+)
+
+// DNSRecordSpec contains the details of a DNS record.
+type DNSRecordSpec struct {
+  // <snip>
+
+  // managementPolicy denotes the current policy applied on the dns record.
+  // Records that have policy set as "Unmanaged" are ignored by the ingress
+  // operator and should be managed at the discretion of the cluster admin.
+  //
+  // +kubebuilder:default:="Managed"
+  // +kubebuilder:validation:Optional
+  // +kubebuilder:validation:Enum=Managed;Unmanaged
+  // +optional
+  ManagementPolicy DNSManagementPolicy `json:"managementPolicy"`
+}
+
+// DNSManagementPolicy is a policy for configuring how the dns controller 
+// manages DNSRecords.
+//
+// +kubebuilder:validation:Enum=Managed;Unmanaged
+type DNSManagementPolicy string
+
+const (
+    // ManagedDNS configures the dns controller manage the lifecyle of the
+    // dns record on the appropriate platform.
+    ManagedDNS DNSManagementPolicy = "Managed"
+    // UnmanagedDNS configures the dns controller to ignore the dns record
+    // and allows to be manually managed by the cluster admin.
+    UnmanagedDNS DNSManagementPolicy = "Unmanaged"
+)
+```
+
+Re-uses existing condition `DNSManaged` of the controller to denote if
+the wildcard *DNSRecord* associated with the controller is managed/unmanaged. 
+
+### Implementation Details/Notes/Constraints
+
+Based on `.loadBalancer.dns`, the controller decides whether to ensure creating
+a wildcard *DNSRecord*. If the customer is updating `.loadBalancer.dns` from 
+`"Managed"` to `"Unmanaged"` the previously created *DNSRecord* will be updated
+appropriately.
+
+The existing state of the ingresscontroller will be synced to the *DNSRecord* as
+well, to clearly state which dns record is unmanaged. The cluster admin can choose
+to delete the *DNSRecord* at his own discretion. 
```

Comment:
```suggestion
The existing `.loadBalancer.dns` value of the ingresscontroller will be replicated to the *DNSRecord*'s `.spec.managementPolicy` as well, to clearly state which dns record is unmanaged. The cluster admin can choose to delete the *DNSRecord* at his own discretion. 
```

This seemed like what the intent of the wording was but feel free to adjust/ignore if that's incorrect. I assumed that what you were trying to state was that the management policy of the IC will be replicated to the management policy of the corresponding DNSRecord.